client/i18n/locales/english/intro.json

```diff
@@ -1814,7 +1814,12 @@
       "rfiq": { "title": "77", "intro": [] },
       "tcwi": { "title": "78", "intro": [] },
       "uptf": { "title": "79", "intro": [] },
-      "nbwu": { "title": "80", "intro": [] },
+      "lab-poker-hand-page": {
+        "title": "Build a Poker Hand Page",
+        "intro": [
+          "For this lab, you will use flexbox to create a webpage of a poker hand."
```

Comment:
maybe we should add a sentence explaining what poker is just in case there are campers that don't know.

I stole this from chat gpt

```suggestion
          "For this lab, you will use flexbox to create a webpage of a poker hand. Poker is a card game that combines skill, strategy, and luck, where players bet on the strength of their hands to win chips or money from others."
```

Reply:
I added a user story "You should build a webpage that displays at least three playing cards."

I think this is enough. If someone wants to make some other types of cards, I think that's fine. Perhaps this should be renamed, poker is often associated with gambling. What do you think about "Build a Hand of Cards"